packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -51,13 +51,13 @@ export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBox
       padding: 0,
       selectors: {
         '&:hover .ms-Button': {
-          backgroundColor: palette.neutralLighter
+          backgroundColor: semanticColors.buttonBackground
```

Comment:
...same here. Plus this slot recently changed to `palette.white` so it's already wrong color. Should be `buttonBackgroundHovered` most likely.

Reply:
i'm surprised it's not a floating X, with a transparent background... unless our redlines have changed since then